plugins/source/okta/codegen/recipes/applications.go

```diff
@@ -0,0 +1,64 @@
+package recipes
+
+import (
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/okta/okta-sdk-golang/v2/okta"
+)
+
+func Applications() []*Resource {
+	return []*Resource{
+		{
+			DataStruct: &okta.Application{},
+			PKColumns:  []string{"id"},
+			Service:    "applications",
+			SkipFields: []string{"Profile"},
+			ExtraColumns: []codegen.ColumnDefinition{
+				{
+					Name:     "profile",
+					Type:     schema.TypeJSON,
+					Resolver: `schema.PathResolver("Profile")`,
+				},
+			},
+			Relations: []string{"ApplicationUsers()", "ApplicationGroupAssignments()"},
+		},
+		{
+			DataStruct: &okta.AppUser{},
+			PKColumns:  []string{"app_id", "id"},
+			SkipFields: []string{"Profile"},
```

Comment:
Is it required? I thought by default `interface{} was transformed to a JSON field and the MarshalJSON would do the magic?

Reply:
Profile was a useless field but I don't remember why.

Reply:
You added it throught extra columns, so, not as useless?

Reply:
tested: Removing `Profile` from here removes it from the generated code. So this looks to be necessary to override it to be JSON.